cmds/wget/wget.go

```diff
@@ -5,37 +5,73 @@
 // Wget reads one file from a url and writes to stdout.
 //
 // Synopsis:
-//     wget URL
+//     wget [ARGS] URL
+//
+// Description:
+//     Returns a non-zero code on failure.
+//
+// Options:
+//     -O: output filename, defaults to '-' (stdout)
```

Comment:
eek. Let's not bring  in that - notation. stdout has a name on modern operating systems :-)
You can do this:
wget_linux.go has
const stdoutName = "/dev/fd/0"

then as we get to other kernels we can work this out. But no more of these names that are not names :-)